Tasks/ServiceFabricDeploy/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 1,
         "Minor": 5,
-        "Patch": 0
+        "Patch": 1
```

Comment:
ServiceFabricComposeDeploy also needs to be bumped up as it uses common helpers.

Reply:
Good catch, the bumps to the other tasks got lost in when merging form master.